src/app/compare/compare-tree/compare-tree.component.css

```diff
@@ -1,3 +1,18 @@
 #tree-container {
   background: linear-gradient(#e9ecef, #e9ecef) no-repeat center/2px 100%;
 }
+
+.compare-header {
+  display: flex;
+  align-items: center;
+  height: 5vh;
+  border-bottom: 1px solid #dee2e6;
+}
+
+.trees-container {
+  height: 46vh;
```

Comment:
Of was het om deze twee dingen op te delen?

Dan adviseer ik toch eerder om een flexbox te gebruiken met flex-direction: column;